src/elements/Badge/Badge.tsx

```diff
@@ -1,41 +1,87 @@
 /* (c) Copyright Frontify Ltd., all rights reserved. */
 
 import { IconProps, IconSize } from "@elements/Icon/Icon";
+import { ReactComponent as Reject } from "@elements/Icon/Svg/Reject.svg";
 import { merge } from "@utilities/merge";
 import { cloneElement, FC, PropsWithChildren, ReactElement } from "react";
 
-export enum BadgeStatus {
+export enum Status {
     Positive = "Positive",
     Progress = "Progress",
     Warning = "Warning",
     Danger = "Danger",
 }
 
-const statusClasses: Record<BadgeStatus, string> = {
-    [BadgeStatus.Positive]: "bg-green-60",
-    [BadgeStatus.Progress]: "bg-violet-60",
-    [BadgeStatus.Warning]: "bg-yellow-60",
-    [BadgeStatus.Danger]: "bg-red-60",
+const statusClasses: Record<Status, string> = {
+    [Status.Positive]: "bg-green-60",
+    [Status.Progress]: "bg-violet-60",
+    [Status.Warning]: "bg-yellow-60",
+    [Status.Danger]: "bg-red-60",
+};
+
+export enum Style {
+    Primary = "Primary",
+    Positive = "Positive",
+    Progress = "Progress",
+    Warning = "Warning",
+    Danger = "Danger",
+}
+
+const styleClasses: Record<Style, string> = {
+    [Style.Primary]: "bg-black-5 text-black-90 dark:text-black-10",
+    [Style.Positive]: "bg-green-20 text-green-90 dark:text-green-50",
+    [Style.Progress]: "bg-violet-20 text-violet-90 dark:text-violet-50",
```

Comment:
The value for `violet-20` in the tailwind config (#e3d4f6) is different from the value in Figma (#E3E8F6). Maybe ask Johanna/responsible person which one to use and adjust accordingly

Reply:
Also, the dark theme version is `violet-40` in Figma

Reply:
41d3718

Reply:
thx for the color "bug" 👍  didn't notice the color difference 🙈  figma is single source of truth when it comes to colors and typography, that's why i  just the colors in the tailwind config